common/reviews/api/typings-generator.api.md

```diff
@@ -57,7 +57,7 @@ export class TypingsGenerator {
     registerDependency(target: string, dependency: string): void;
     // (undocumented)
     runWatcherAsync(): Promise<void>;
-}
+    }
```

Comment:
Is this a bug in api extractor?

Reply:
Maybe. @octogonz?

Reply:
Yes. Lemme see what's up

Reply:
Here's an isolated repro: https://github.com/microsoft/rushstack/issues/2792